Mathlib/Analysis/SpecialFunctions/Gamma/Beta.lean

```diff
@@ -458,10 +458,95 @@ theorem Gamma_ne_zero_of_re_pos {s : ℂ} (hs : 0 < re s) : Gamma s ≠ 0 := by
   contrapose! hs
   simpa only [hs, neg_re, ← ofReal_natCast, ofReal_re, neg_nonpos] using Nat.cast_nonneg _
 
+lemma integral_re {X : Type*} [MeasurableSpace X]
+    (μ : Measure X) {f : X → ℂ} (hf : Integrable f μ) :
+    ∫ x, (f x).re ∂μ = (∫x, f x ∂μ).re := _root_.integral_re hf
+
+lemma setIntegral_re {X : Type*} [MeasurableSpace X]
+    (μ : Measure X) {s : Set X} {f : X → ℂ} (hf : IntegrableOn f s μ) :
+    ∫ x in s, (f x).re ∂μ = (∫x in s, f x ∂μ).re := Complex.integral_re _ hf.integrable
+
```

Comment:
You should open a separate PR for these, as they have nothing to do with the `Beta` function, and make this PR depend on it (the hidden comment in the PR description explains how to indicate a dependency). I would put them in `Mathlib.MeasureTheory.Integral.SetIntegral`, along their `RCLike` versions. Please also add an `intervalIntegral` version.